components/SpacesBottomSheet/index.tsx

```
@@ -41,30 +42,34 @@ const SpacesBottomSheet = React.forwardRef<
-
   // renders
   const renderFooter = useCallback(
-    (props: any) => (
-      <BottomSheetFooter {...props} style={{ backgroundColor: 'black' }}>
-        {activeLivekitRoom && (
-          <SpacesBottomControls isHost={activeLivekitRoom?.is_host || false} />
-        )}
-      </BottomSheetFooter>
-    ),
+    (props: any) => {
+      const insets = useSafeAreaInsets();
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)